content/chainguard/chainguard-enforce/iam-groups/identity-examples/enforce-gitlab-identity.md

```diff
@@ -355,7 +355,7 @@ assume-and-explore:
 
 Let's go over what this configuration does. 
 
-First, GitLab requires that pipelines have a shell. To this end, this configuration uses the [`cgr.dev/chainguard/wolfi-base` image](https://edu.chainguard.dev/chainguard/chainguard-images/reference/wolfi-base/overview/) since it includes the `sh` shell.
+First, GitLab requires that pipelines have a shell. To this end, this configuration uses the [`cgr.dev/chainguard/wolfi-base` image](https://edu.chainguard.dev/chainguard/chainguard-images/reference/wolfi-base/) since it includes the `sh` shell.
```

Comment:
I searched for this broken link also, this was the only one I found besides the blurb